Source/Core/DolphinQt/Config/PatchesWidget.cpp

```diff
@@ -145,8 +145,18 @@ void PatchesWidget::SavePatches()
 
     for (const auto& entry : patch.entries)
     {
-      lines.emplace_back(StringFromFormat("0x%08X:%s:0x%08X", entry.address,
-                                          PatchEngine::PatchTypeAsString(entry.type), entry.value));
+      if (!entry.conditional)
+      {
+        lines.emplace_back(StringFromFormat("0x%08X:%s:0x%08X", entry.address,
```

Comment:
We should probably use [fmt](https://fmt.dev/latest/index.html) for new format calls.
```suggestion
        lines.emplace_back(fmt::format("{:#10X}:{}:{:#10X}", entry.address,
```
(`#include <fmt/format.h>` first)